giles-eco/src/main/java/edu/asu/diging/gilesecosystem/web/service/properties/Properties.java

```diff
@@ -49,4 +52,6 @@
     public final static String GILES_TMP_FOLDER = "giles_files_tmp_dir";
 
     public final static String FREDDIE_HOST = "freddie_host";
+
+    public final static String AUTHORIZATION_TYPE = "accessToken";
```

Comment:
this doesn't belong here. ```Properties``` holds keys that can be found in the properties file; either put the values in the properties file or create a new interface for authorization types;

Also, ```AUTHORIZATION_TYPE``` is a bad name, it doesn't say anything about the type of authorization which is the whole point of this.